style/themes/lcars.css

```diff
@@ -231,6 +231,24 @@ th {
   background: transparent;
 }
 
+/***style the API token modal close button***/
+.modal-header .close,
+button.close {
+  background: #dd4b39;
+  padding: 0 0.5em 0 1em;
+  color: #000;
+  opacity: 1;
+  text-decoration: none;
+  border-radius: 1.07em;
+  text-align: right;
+}
+
+.modal-header .close:hover,
+button.close:hover {
+  background: #d73925;
+  color: #fff;
+}
+
```

Comment:
These lines are changing the same elements changed on lines 1511-1521.

Please move this block to line 1511 merge the `.close` styles.

Reply:
Done. I also noticed that in doing this it changed the alert close button, so I modified it to better match this style. It was difficult to see before though, so this may be a good thing, but I won't be offended if people don't like it:
![image](https://user-images.githubusercontent.com/13670462/226794166-aaef25ba-2753-435c-a3c0-0ca820e737dd.png)
![image](https://user-images.githubusercontent.com/13670462/226794315-705adec2-b440-49a9-a7cc-b205821d2f07.png)

Reply:
I think the close button doesn't need a background.

Can you try a different approach, using a larger "x" and no background?

```css
.close,
.alert > .close {
  color: currentColor;
  opacity: 1;
  text-shadow: none;
  transform: scale(2) translate(0, -3px);
  font-weight: normal;
}

.close:hover,
.close:focus {
  color: #fff;
  opacity: 1;
}
```
![image](https://user-images.githubusercontent.com/1385443/226972209-882a36e3-e9fe-4590-aa87-251b7937e303.png)

Reply:
That looks good to me. I updated the script.